### DISCUSSION TITLE
Raspiblitz sync problems

### DISCUSSION THREAD

**Discussion by domodomodomo3** [raspiblitz/raspiblitz#4229] [Q&A]
trying to figure out whats from with my node. Since the update i cant get it working again.
it was stuck in a loop of synching restarts.
since i couldnt figure out what to do i started fresh. deleteted everything ssd and sd.
but still problems. maybe some hardware is broken?

here are my logs:
https://l.termbin.com/d34t

thanks for your help

**Comment by rootzoll** [comment]
Yes I can see the bitcoind restarting in the logs because of an error - but the debug.logs are to short of an snippet to see why. You may need to repost mor of this: `sudo journalctl -u bitcoind -b --no-pager -n200`
